### DISCUSSION TITLE
Is there a documentation about using CEL in AuthConfig ?

### DISCUSSION THREAD

**Discussion by eshepelyuk** [Kuadrant/authorino#504] [Q&A] [answered]
The question is title.

**Comment by guicassolato** [comment]
Only in the CRD, I'm afraid. But we definitely need some for GitHub and the public website too.

**Reply by guicassolato** [reply, marked as answer]
In the meantime...
1. Where previously one could do `{selector: <json-path>, operator: <authorino-custom-operator>, value: <string>}`, now `{predicate: <cel-predicate>}` is also supported, where `<cel-predicate>` has to evaluate to a boolean value;
2. Where previously one could do `{selector: <json-path}`, now `{expression: <cel-expression>}` is also supported;
3. Specific fields that supported [JSON path interpolation](https://docs.kuadrant.io/0.11.0/authorino/docs/features/#interpolation) (e.g. `http.url: <string>`) have got a CEL-enabled counterpart (e.g. `http.urlExpression: <cel-expression>`).

All CEL predicates and expressions must be valid or the AuthConfig will be rejected (at reconciliation time), unlike JSON paths that at runtime will evaluate to null/empty value if invalid.

**Reply by eshepelyuk** [reply]
Thnx @guicassolato 
Would be great to have all this in `User guides`.
